metricbeat/mb/testing/modules.go

```diff
@@ -269,7 +269,10 @@ func RunPushMetricSetV2(timeout time.Duration, waitEvents int, metricSet mb.Push
 			select {
 			case <-timer.C:
 				return
-			case e := <-r.eventsC:
+			case e, ok := <-r.eventsC:
```

Comment:
@andrewkroh Is this change related? LGTM but curious if that is related to an other issue.

Reply:
While testing some failure conditions I noticed the test case didn’t stop immediately and this was the fix.